plat/xen/include/xen/io/protocols.h

```diff
@@ -29,9 +29,7 @@
 #define XEN_IO_PROTO_ABI_X86_64     "x86_64-abi"
 #define XEN_IO_PROTO_ABI_ARM        "arm-abi"
 
-#if defined(__i386__)
```

Comment:
The headers within `plat/xen/inckude/xen` should not be touched. These are a copy from upstream Xen that are updated/synced with Xen.

Reply:
Thanks for the review, I will restore these headers in the next commit.

Reply:
Hey, Does this mean that I should only change the headers within the file xen-x86/mm.h since this is the only file that contains __i386__ headers and is not under the directory plat/xen/include/xen.